obelisk/python/obelisk_py/core/utils/launch_utils.py

```diff
@@ -317,3 +319,30 @@ def _single_component_launch_actions(node_settings: Dict, suffix: Optional[int]
         return node_launch_actions
     else:
         return _single_component_launch_actions(node_settings)
+
+
+def setup_logging_dir(config_name: str) -> str:
+    """
+    Configures the logging directory.
+
+    Checks if there is already an obk_logs directory, and if not, create on.
+    Then creates a folder inside of that will the date and the name of the config.
+    Sets the ROS_LOG_DIR environment variable to this location.
+    """
+    # Check for the directory
+    general_log_file_path = os.getcwd() + "/obk_logs"
```

Comment:
need to be careful - `getcwd()` will point to the directory where you launch, which might not be in the repo root. if you want the logs to be located somewhere specific, we need to set the `ROS_HOME` variable

Reply:
What would you suggest setting `ROS_HOME` to? Seems like you run into the same issue which is that we don't know the path to the user's workspace. So for now I'm just assuming that is where they launched from.